lib/markdownlint.js

```diff
@@ -895,6 +895,39 @@ function parseConfiguration(name, content, parsers) {
   };
 }
 
+/**
+ * Resolve referenced "extends" path in a configuration file
+ * using path.resolve() and require.resolve() as a fallback.
+ *
+ * @param {string} configFile Configuration file name.
+ * @param {string} referenceId Referenced path to resolve.
+ * @returns {string} resolved path to file
+ */
+function resolveConfigExtends(configFile, referenceId) {
+  const configFileDirname = path.dirname(configFile);
+  const resolvedExtendsFile = path.resolve(configFileDirname, referenceId);
+
+  try {
+    if (fs.statSync(resolvedExtendsFile).isFile()) {
+      return resolvedExtendsFile;
+    }
+  // eslint-disable-next-line unicorn/prefer-optional-catch-binding
+  } catch (error) {
+    // If fs.statSync has thrown, trying require.resolve
+  }
+
+  try {
+    return require.resolve(referenceId);
+
+  /* c8 ignore next 2 */
```

Comment:
I had to add this line after facing quite a mystery. Without it, running `npm run ci; npx c8 report --reporter html` _on Node 12_ results in this:

<img width="642" alt="Screenshot 2020-10-18 at 12 43 30" src="https://user-images.githubusercontent.com/608862/96366479-b3145c80-113f-11eb-89c7-abac1b8f84ff.png">

This is a false-positive because both branches are covered. If I add `console.log` in both, they do show up. Other versions though [don’t suffer from this problem](https://github.com/DavidAnson/markdownlint/runs/1270965501). Thoughts?

Reply:
C8 only seems to be reliable in the LATEST Node (14). In fact, it already does not run in the CI workflow for this project for Node 10. On another project of mine, it ONLY runs on Node 14. If the behavior you're seeing here is correct on Node 14, we can tweak `nodejs.yml` to do the same here.

Reply:
I tweaked `nodejs.yml` as you suggested, CI passed. Thanks! Resoving.